web/static.go

```diff
@@ -87,17 +87,22 @@ func staticFilesHandler(handler http.Handler) http.Handler {
 	})
 }
 
-func staticFilesWithValidationHandler(handler http.Handler) http.Handler {
-	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		// Require validation from any cache, achieved via Last-Modified and the
-		// http.FileServer.
+type notFoundNoCacheRespWr struct {
```

Comment:
nit: is it worth abbreviating `RespWr`? I might suggest going for verbose given the minimal impact to the code and increased readability.